src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball:\nColor: " + color + "\nNumber of ball: " + number;
+    }
```

Comment:
I'd recommend generating toString method using Intellij IDEA tools to make it more clear and consistent